pkgs/development/python-modules/pytest-httpserver/default.nix

```diff
@@ -11,22 +11,14 @@
 
 buildPythonPackage rec {
   pname = "pytest-httpserver";
-  version = "0.3.6";
+  version = "0.3.7";
 
   src = fetchPypi {
     pname = "pytest_httpserver";
     inherit version;
-    sha256 = "1wdhbzv6x2v4qsqwgsc5660c4lxplh9b61vfj1zqhbhs36y96vl9";
+    sha256 = "sha256-YgTcrUlwh2jz0tJdMUgjm8RcqrtpJ/oUQm3SnxUc5Z4=";
   };
 
-  patches = [
-    (fetchpatch {
```

Comment:
Please remove fetchpatch from inputs.

Reply:
Thanks a lot for the feedback.

Removed.